pkg/eventcounter/api/api_test.go

```diff
@@ -786,6 +786,106 @@ func TestGetMAUCount(t *testing.T) {
 	}
 }
 
+func TestSummarizeMAUCounts(t *testing.T) {
```

Comment:
Can you also add the tests for `Unauthenticated`, `PermissionDenied`, and `Internal` while calling the `checkAdminRole`?

Reply:
@cre8ivejp 
Thanks, I added it 👍